Gemfile.tt

```diff
@@ -9,6 +9,7 @@ gem 'mini_magick' # A ruby wrapper for ImageMagick or GraphicsMagick command lin
 gem 'pagy' # A pagination gem that is very light and fast
 gem 'discard' # Soft deletes for ActiveRecord
 gem 'sidekiq' # background processing for Ruby
+gem 'sassc' # bootsnap dependency
```

Comment:
I'm sorry about this comment, which is a bit late but I'm not sure about this gem, it's an old gem (last update was 2020) and I remember SASS in C [was deprecated](https://sass-lang.com/blog/libsass-is-deprecated), the SASS team moved to the DartSass already, and we don't use this gem in the generated project at all....

Maybe this gem is needed during generating the new project, but later on we don't use it, should we remove it at the end? with a post setup callback? cc @sanG-github

Reply:
I agree with the above statement, but it would be best to move this concern into an issue to address it in the next release.

Reply:
That's appropriate, but as far as I remember, we still need it to run the project, not only generating it.
In that case, I think we need to use another alternative. (Note that this one was caused by the bootsnap gem)

Reply:
Maybe a solution is just creating a new issue to remove this gem :D so all good with this release cc @malparty

Reply:
This is the actual bug fix that prevented to use the Template :harold: 

yes, I'll create an issue 🙇

Reply:
✅  https://github.com/nimblehq/rails-templates/issues/416